shapeshift/web
# feat: foxpage header and tabs responsive

## Description
- As we now have responsive mockups, we need to adapt the work we did to new mockups
- I switched the tab behavior to a manual one
- I used a `Menu` component including some tabs too
- I changed the `FoxTab` mobile style
- Fixed some wrong padding added to app code
- Added this responsive behavior to app code

The selected asset logic should be done in application code and after the architecture planning so we know what to do, but at least the tab logic works inside the storybook story



## Notice




## Pull Request Type


## Issue (if applicable)



## Risk
#1766 #1765 
- No risk for the moment, inside storybook stories


## Testing
- Go to `FoxPage` in the app, and see if it looks fine on desktop
- Use the mobile tool of storybook to check on small, large mobile
![image](https://user-images.githubusercontent.com/14963751/169073118-8fe95fd7-d870-4dd8-842f-361e23da8593.png)
- See if the component is a dropdown and if selecting an asset switch the `TabPanels` content
- I also removed the background of the header component on mobile as it's not needed anymore


## Screenshots (if applicable)
<img width="296" alt="image" src="https://user-images.githubusercontent.com/14963751/169622311-99a1a275-9371-4ae9-b036-53050a3bda85.png">
<img width="1242" alt="image" src="https://user-images.githubusercontent.com/14963751/169622337-9a5a1039-6538-4d2c-9ee1-e9d623690c73.png">
